Auth/OpenID/Parse.php

```diff
@@ -218,12 +218,12 @@ function removeQuotes($str)
     
     function match($regexp, $text, &$match)
     {
-        if (!is_callable('mb_ereg_search_init')) {
-            if (!preg_match($regexp, $text, $match)) {
-                return false;
-            }
-            $match = $match[0];
```

Comment:
Did my comment https://github.com/openid/php-openid/pull/134#issuecomment-287699942 address your concern ? You can check by yourself that it is *expected* that the `match` function returns an array, at least from the function used to extract links (openid rel) from the page.